web-admin/src/features/projects/environment-variables/ValueCell.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+  import Tooltip from "@rilldata/web-common/components/tooltip/Tooltip.svelte";
+  import TooltipContent from "@rilldata/web-common/components/tooltip/TooltipContent.svelte";
+  import { EyeIcon, EyeOffIcon } from "lucide-svelte";
+
+  export let value: string;
+
+  let showValue = false;
+  let copied = false;
+  let displayValue = value;
+
+  $: inputType = showValue ? "text" : "password";
+  $: isEmpty = value.length === 0;
+  $: isValueHidden = !showValue;
+
+  // 16 characters
+  const REDACTED_VALUE = "****************";
+
+  $: if (!showValue) {
+    displayValue = REDACTED_VALUE;
+  } else {
+    if (isEmpty) {
+      displayValue = "Empty";
+    } else {
+      displayValue = value;
+    }
+  }
+
+  function toggleShowValue() {
+    showValue = !showValue;
+  }
+
+  function copyToClipboard(text: string) {
+    navigator.clipboard.writeText(text).catch(console.error);
+  }
```

Comment:
What about the `copyToClipboard` function in `web-common/src/lib/actions/copy-to-clipboard.ts`?